doc/articles/faq.md

```diff
@@ -0,0 +1,268 @@
+
+## Error `targets 'netstandard2.0'. It cannot be referenced by a project that targets 'UAP,Version=v10.0.14393'`
+	
+```
+Cannot resolve Assembly or Windows Metadata file 'Type universe cannot resolve assembly: netstandard, Version=2.0.0.0, Culture=neutral, [...]
+
+System.IO.FileLoadException: Could not load file or assembly 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=[...]'. The located assembly's manifest definition does not match the assembly reference.
+```
+
+Make sure to upgrade your SDK version and nuget packages. You have to target Fall Creators update or later and reference the netcore package to 6.0+.
+
+## when I navigate this ways `_frame.Navigate(typeof(MainPage));` I never call MainPage ctor. How to inject MainPage datacontext then?
+
+The `DataContext` can be passed as a navigation parameter, then set as `DataContext` in the `OnNavigatedTo` method
+
+## I'm trying to ref a net standard library into my iOS project but it says "The project(s) MyApp.Services.Interfaces did not provide any metadata reference".
+
+It's an existing issue, caused by Roslyn. You must add all your TargetFrameworks to your netstd2.0 project, and use Oren's MSBuid.Extras for that [source generation issue](https://github.com/nventive/Uno.SourceGeneration/issues/2)
+
+# Can Uno.UI convert a uwp control to native android/ios and also has the option to use android native controls within xaml?
+
+That's exactly right, see [this](https://github.com/nventive/Uno/blob/a69879a3154f61b2d493be433aa08bc3b8aa1b06/src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml#L2843) for the XAML-UWP button, and [this](https://github.com/nventive/Uno/blob/a69879a3154f61b2d493be433aa08bc3b8aa1b06/src/Uno.UI/UI/Xaml/Style/Generic/Generic.Native.xaml#L20) for the iOS native button When complete XAML is used for rendering, it's not so much a conversion but more of  a vector rendering. For instance, Uno.UI.iOS uses CGLayer for rendering content it's different because Uno.UI integrates within the layouting system of the platform, which allows for mixed rendering. Uno.UI supports webviews, whereas Flutter does not. Flutter also has to render everything, including what the platform provides by default, such as the Magnifier in the TextBox for accessibility (they is not yet supported either).
+
+## Can I have a screen and use a Map and put markers, using UNO, for Android, and Windows 10?
+> And can I access the camera, take a picture and Post to a REST WebAPI? can I access Photos / files in phone?
+	
+For now, we have internal code to do all of that, but in the meantime, you have to do this by hand on each. Parts of https://docs.microsoft.com/en-us/xamarin/essentials/ allows to do it.
+
+## The PointerReleased event is NOT fired for the Border control on Android
+
+There's a bug on Android where `PointerReleased` is not fired if `PointerPressed` is not handled.
```

Comment:
It's a bug or a feature?

Reply:
It is a bug because it's not what windows does.